src/main/kotlin/no/nav/syfo/client/ArenaDialogNotat.kt

```diff
@@ -136,7 +136,9 @@ fun sendArenaDialogNotat(
     session: Session,
     arenaDialogNotat: ArenaDialogNotat,
     loggingMeta: LoggingMeta
-) = producer.send(session.createTextMessage().apply {
-    text = arenaDialogNotatMarshaller.toString(arenaDialogNotat)
-    log.info("Message is sendt to arena {}", fields(loggingMeta))
-})
+) = producer.send(
+    session.createTextMessage().apply {
+        text = arenaDialogNotatMarshaller.toString(arenaDialogNotat)
+        log.info("Message is sendt to arena {}", fields(loggingMeta))
```

Comment:
Kanskje rette til "Message is about to be sent.." siden dette vil logges *før* meldingen sendes.

Reply:
Det er sikkert lurt! Det er kun linting i denne oppgaven, så jeg lager en lapp på det 👍